File: backend/src/main/scala/kpn/api/common/SurveyDateInfo.scala
package kpn.api.common

import kpn.api.custom.Day

case class SurveyDateInfo(
  now: Day,
  lastWeekStart: Day,
  lastMonthStart: Day,
  lastHalfYearStart: Day,
  lastYearStart: Day,
  lastTwoYearsStart: Day
)

File: backend/api/src/main/java/kpn/api/common/SurveyDateInfo.java
package kpn.api.common;

import kpn.api.custom.Day;

public record SurveyDateInfo(
  Day now,
  Day lastWeekStart,
  Day lastMonthStart,
  Day lastHalfYearStart,
  Day lastYearStart,
  Day lastTwoYearsStart
) {
}
